index.js

```diff
@@ -8,24 +8,23 @@ var TrackerGroup = exports.TrackerGroup = function (name) {
   EventEmitter.call(this)
   this.name = name
   this.trackGroup = []
-  var self = this
   this.totalWeight = 0
-  var noteChange = this.noteChange = function (name) {
-    self.emit("change", name || this.name)
-  }.bind(this)
-  this.trackGroup.forEach(function(unit) {
-    unit.on("change", noteChange)
-  })
+  this.noteChange = TrackerGroup.prototype.noteChange.bind(this);
```

Comment:
It's actually [not anymore](https://chromium.googlesource.com/v8/v8/+/97def8070cbb979419c041c3732d5e11779f79da) (I don't think this has hit Node yet), but I agree, could just make it a closure so it's fast on older engines too.